### ISSUE TITLE
Version solving failed

### ISSUE THREAD

**Issue by KMZabirTarique** [zeeshux7860/ImageEditorPro#35]
hello,
I am developing an application, which contains  this libraries-

dependencies:
  flutter:
    sdk: flutter
  bloc: ^6.0.3
  flutter_bloc: ^6.0.6
  equatable: ^1.0.2
  cupertino_icons: ^1.0.2
  buttons_tabbar: ^1.2.1
  tab_indicator_styler: ^2.0.0-null-safety
  contained_tab_bar_view: ^0.7.1
  speech_recognition: ^0.3.0+1
  table_calendar: ^3.0.0
  flutter_week_view: ^1.1.0
  md2_tab_indicator: ^1.0.2
  expandable: ^5.0.1
  google_maps_flutter: ^2.0.3
  http: ^0.12.2
  encrypt: ^4.1.0
  file_picker: ^3.0.1
  archive: ^2.0.13
  path_provider:
  url_launcher:
  pbkdf2_dart: ^2.1.0
  dio: ^3.0.10
  aes_crypt: ^0.1.1
  intl: ^0.17.0
  global_configuration: ^1.6.0
  imei_plugin: ^1.2.0
  device_info: ^2.0.0
  sqflite: ^2.0.0+3
  screenshot: ^0.3.0
<img width="915" alt="1" src="https://user-images.githubusercontent.com/77930219/121805370-d9f24680-cc6c-11eb-841d-fe41a382d2e6.png">
<img width="290" alt="2" src="https://user-images.githubusercontent.com/77930219/121805372-db237380-cc6c-11eb-9ca0-bf682ab54f80.png">
<img width="246" alt="4" src="https://user-images.githubusercontent.com/77930219/121805375-dbbc0a00-cc6c-11eb-83cc-e96583c658cd.png">
<img width="915" alt="1" src="https://user-images.githubusercontent.com/77930219/121805386-e70f3580-cc6c-11eb-85aa-1a33b3ec8503.png">

  camerawesome: ^0.2.2
  path:
  video_player:
  shared_preferences: ^2.0.6
  speech_to_text: ^4.2.1
  image_editor_pro: ^1.1.8

while I import image_editor_pro: ^1.1.8 in the yaml file it seems something like this-
Because image_editor_pro >=1.1.0 depends on dio ^4.0.0 and test_app depends on dio ^3.0.10, image_editor_pro >=1.1.0 is forbidden.
So, because test_app  depends on image_editor_pro ^1.1.8, version solving failed.
pub get failed (1; So, because test_app  depends on image_editor_pro ^1.1.8, version solving failed.)

how can I fix this error. I need to use image editor in my app. Is there any way to solve this issue?
Thank you in advance.

**Comment by AshfaqAshi** [comment]
Yes, it uses old versions of dependencies and version solving fails if we use new versions of those packages

**Comment by KMZabirTarique** [comment]
Thanks for your information. Can you please tell me is there any way to resolve it?

**Comment by KMZabirTarique** [comment]
> Not from your side, the package should be updated to a new version.

Thanks a lot.
